Dockerfile

```diff
@@ -50,33 +50,26 @@ RUN curl -L -o- https://download.docker.com/linux/static/stable/x86_64/docker-$D
     chmod 755 /usr/local/bin/docker
 
 ARG KUBE_VERSION=1.19.7
-RUN curl -L -o /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v$KUBE_VERSION/bin/linux/amd64/kubectl && \
+RUN curl -L -o /usr/local/bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v$KUBE_VERSION/bin/linux/$(dpkg --print-architecture)/kubectl && \
     chmod 755 /usr/local/bin/kubectl
 
 ARG HELM_VERSION=3.5.2
```

Comment:
I wanted to upgrade this to 3.6 but we've gotten bit before by helm upgrades - so did not put that here